docs/components/buttons-menus/links.md

```diff
@@ -21,7 +21,10 @@ If the action taken by the user will change or manipulate data, use a [button](b
 
 ### States
 
-Links support the different styles: default/visited, hover, pressed, focused and disabled
+Links support the different styles: default/visited, hover, pressed, focused and disabled.
+
+Links inside a paragraph of text need the class `.link-text` to be accessible by always being
+underlined.
```

Comment:
@dr-itz This should be moved to the code section, as it talks about the implementation.